### DISCUSSION TITLE
gestire API GET in vb6

### DISCUSSION THREAD

**Discussion by luca2009** [opendatasicilia/opendatasicilia.it#72] [Domande]
Come faccio la get in vb6 per ricevere in dati con:
https://github.com/opendatasicilia/comuni-italiani/blob/main/api/README.md
grazie.

**Comment by aborruso** [comment]
Ciao @luca2009 , non conosco vb6.

In generale ti consiglio sempre di aggiungere a domande come queste, anche qualche dettaglio sui test che hai già fatto, cosa non ha funzionato, che strumenti hai usato, ecc..
